.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same with this. ignore the `.idea` directory entirely

Reply:
I will make an update with the `.idea` ignored